web-admin/src/features/navigation/nav-utils.ts

```diff
@@ -18,3 +19,26 @@ export function isDashboardPage(page: Page): boolean {
 export function isReportPage(page: Page): boolean {
   return page.route.id === "/[organization]/[project]/-/reports/[report]";
 }
+
+export function isReportExportPage(page: Page): boolean {
+  return (
+    page.route.id ===
+    "/[organization]/[project]/[dashboard]/-/reports/[report]/export"
+  );
+}
+
+export function getScreenNameFromPage(page: Page): MetricsEventScreenName {
+  switch (true) {
+    case isOrganizationPage(page):
+      return MetricsEventScreenName.Organization;
+    case isProjectPage(page):
+      return MetricsEventScreenName.Project;
+    case isDashboardPage(page):
+      return MetricsEventScreenName.Dashboard;
+    case isReportPage(page):
+      return MetricsEventScreenName.Report;
+    case isReportExportPage(page):
+      return MetricsEventScreenName.ReportExport;
+  }
+  return MetricsEventScreenName.Home;
```

Comment:
What about `MetricsEventScreenName.Other`? E.g., when we add the Alerts pages, if we forget to add a mapping, then we'd want the telemetry to say "other" not "home".

Reply:
I think unknown might be a better fit.